lib/tasks/db.js

```diff
@@ -297,6 +297,8 @@ function getSequelizeInstance () {
 function getMigrator (type, callback) {
   if (helpers.config.configFileExists() || args.url) {
     var sequelize = getSequelizeInstance();
+    var coffeeRegex = /^(?!skeleton)(.*)\.js$|\.coffee$/;
```

Comment:
yup, I was being redundant there :disappointed: now that you pointed that out I see it clearly :laughing: